crates/ironrdp-cliprdr/src/backend/mod.rs

```diff
@@ -0,0 +1,113 @@
+#[cfg(windows)]
+pub mod windows;
```

Comment:
Did you consider the [`clipboard-win` crate](https://lib.rs/crates/clipboard-win) before implementing your own backend? I’m not sure there is everything in there, and they are still using `winapi` instead of `windows` though.

Reply:
AFIK `clipboard-win` does not allow listening for clipboard changes and does not provide API for delayed rendering, which are must-haves for a correct `CLIPRDR` implementation

Reply:
Makes sense :+1: